src/components/notification/z-pocket-body/styles.css

```diff
@@ -0,0 +1,49 @@
+:host {
+  font-family: var(--dashboard-font);
+  font-weight: var(--font-rg);
+  display: block;
+  width: 100%;
+  box-sizing: border-box;
+}
+
+:host > main {
+  background: var(--bg-white);
+  border-left: var(--border-size-small) solid var(--bg-grey-200);
+  border-right: var(--border-size-small) solid var(--bg-grey-200);
+  overflow: auto;
+  scrollbar-width: none;
+}
+
+:host > main.preview {
+  max-height: 225px;
+  transition: all 200ms ease-in-out, top 200ms ease-in-out;
+}
+
+:host > main.open {
+  max-height: calc(100vh - 48px);
+  transition: all 200ms ease-in-out, top 200ms ease-in-out;
+}
+
+:host > main.closed {
+  height: 0px;
+  transition: all 200ms ease-in-out, top 200ms ease-in-out;
+}
+
+:host ::-webkit-scrollbar {
```

Comment:
occhio che la pseudo `::-webkit-scrollbar` non è supportata da tutti i browser: https://css-tricks.com/almanac/properties/s/scrollbar/ https://stackoverflow.com/questions/6165472/custom-css-scrollbar-for-firefox
Fai un check per vedere come si comporta con la width iniziale

Reply:
Sì sì, se vedi sopra c'è anche un `scrollbar-width: none;` proprio perché funzioni un po' ovunque.